File: Codigos estudiantes por lenguaje/Java/Sebastian_Acevedo/Introducción/Diccionario.java
import java.util.HashMap;

public class Main {

    public static void main(String[] args){
        HashMap<String, String> dicc= new HashMap<>();
        dicc.put("Programar", "Utilizacion de codigo para ejecutar un programa");
        dicc.put("vainazo", "lo que se tiran en el grupo de Discord");
        dicc.put("among us","juego para funar");
        dicc.put("Futbol", "Deporte con una pelota y arcos");
        dicc.put("Funar","Gritar como una loca(Diego cof cof)");
        for (String concepto: dicc.keySet()) {
			String key = concepto;
			String value = dicc.get(concepto);
			System.out.println(key+" = "+value);
		}
    }
}

File: Codigos estudiantes por lenguaje/PY/Acevedo_Sebastian/Otros (Sin ordenar, nn, etc)/diccionario.py
from sys import stdin,stdout
import operator

dicc = {}
dicc['Programar'] = 'Utilizacion de codigo para ejecutar un programa'
dicc['Funar'] = 'Gritar como una loca'
dicc['Futbol'] = 'Deporte con una pelota y arcos'
dicc['Among Us'] = 'Juego para funar'
dicc['Impostor'] = 'Si, ya sabemos'
dicc['Edwin'] = 'Parlon de Miedo'

dicc_ordenado_keys = sorted(dicc.items(),key=operator.itemgetter(0)) #ordenar por las llaves

dicc_ordenado_values = sorted(dicc.items(),key=operator.itemgetter(1)) #ordenar por valores

print(dicc_ordenado_keys)

print(dicc_ordenado_values)

"""
dicc = {}
dicc['Programar'] = 'Utilizacion de codigo para ejecutar un programa'
dicc['Funar'] = 'Gritar como una loca'
dicc['Futbol'] = 'Deporte con una pelota y arcos'
dicc['Among Us'] = 'Juego para funar'
dicc['Impostor'] = 'Si, ya sabemos'
dicc['Edwin'] = 'Parlon de Miedo'

for concepto, definicion in dicc.items():
    print(f'{concepto} -> {definicion}')
"""
